2_process.R

```diff
@@ -24,7 +25,11 @@ p2_targets_list <- list(
   # Create a list of unique site locations containing DO data  
   tar_target(
     p2_site_list_csv,
-    create_site_list(p2_filtered_wqp_data_subset,p1_lowerdrb_boundary,p1_nwis_sites,p1_daily_data,p2_inst_data_hourly,fileout = "2_process/out/DRB_DO_sitelist.csv"),
+    {
```

Comment:
Same comment here as above. I'm thinking we should just make our lives simpler by doing an object target instead of a file one here.

Reply:
I agree with you that making the (discrete data) wqp subset above into an objects file rather than a targets file might be easier to work with. I also assume that to prep the discrete data as modeling inputs, we will need at least one more downstream step to combine with the daily continuous values. So it probably makes more sense for `p2_filtered_wqp_data_subset` to be an object target. 

I'm not as sure on the site list - this table contains the lat/lon of the unique site id's and contains basic summaries/tallies of the available data. The csv does get read in within the Rmd file, but I guess I'm wondering whether this target is useful as a file format so that it can be accessed and viewed outside of the pipeline.

Reply:
That's a good point that this could be quite useful outside the pipeline. Now that you say that, I'm realizing that I am saving the one with the reach_ids to csv in #35. So let's leave the `p2_site_list_csv` as a file target in the PR, but I think it gets superseded by the `p2_sites_w_segs_csv` target in #35

Reply:
Ah, OK. That makes sense that this table is superseded with the target that contains the paired segs.